windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -33,6 +33,16 @@ endif::openshift-origin[]
 
 This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in 
 
+[id="wmco-6-0-0-new-features"]
+=== New features
+
+[id="wmco-6-0-0-containerd"]
+==== Containerd is the default container runtime
+
+Because the Docker runtime is deprecated Kubernetes 1.24, containerD is now the default runtime for WMCO-supported Windows nodes. Upon installation of or upgrade to WMCO 6.0.0, containerd is installed as a Windows service. The kubelet will use containerd for image pulls instead of the Docker runtime. Users no longer need to enable the Docker-formatted container runtime or install the Docker container runtime on Bring-Your-Own-Host (BYOH) instances. You can continue to use nodes based on VM images that use Docker. Containerd can run along with Docker service.
```

Comment:
```suggestion
Because the Docker runtime is deprecated with Kubernetes 1.24, containerD is now the default runtime for WMCO-supported Windows nodes. Upon the installation of or an upgrade to WMCO 6.0.0, containerd is installed as a Windows service. The kubelet now uses containerd for image pulls, instead of the Docker runtime. Users no longer need to enable the Docker-formatted container runtime or install the Docker container runtime on Bring-Your-Own-Host (BYOH) instances. You can continue to use nodes based on VM images that use Docker. Containerd can run along with the Docker service.
```

Reply:
Also not sure if `containerD` s/B `containerd`.

Reply:
@mjpytlak Lower-case d: [containerd](https://containerd.io/)